requirements.txt

```diff
@@ -4,7 +4,7 @@ gevent-websocket==0.10.1
 wsaccel==0.6.3  # recommended for acceleration of gevent-websocket. But abandoned.
 web3==5.21.0
 pysqlcipher3==1.0.4
-requests==2.25.1
+requests==2.26.0
```

Comment:
I already commented in their repo that this is not good. We can't be limited by their requirements and have to upgrade when they do.

https://github.com/polkascan/py-substrate-interface/commit/c9e48460c4cd5b2d00e5e717eecba4b581e3c166#r53677476

Reply:
Do we still want to upgrade requests here?

Reply:
yes it's okay. But need to test in case it breaks anything.